www/shards/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        crystal-lang shards 0.8.1-devel v
+
+categories          www
```

Comment:
By the way, I only just realized, is this actually www specific? If not, maybe it belongs in another category?

Reply:
This port is applicable to Crystal Programming Language and its web development framework called `Amber`.  Next, this port would be in similar categories as the `rb-bundler` port.  Thus, it would it maybe should be something like `crystal, www, devel`.  Would this be better?

Reply:
Probably, yes, though we don't yet have a `crystal` category. That said, if we get enough crystal code, one would be appropriate. Certainly `devel` would be a good addition. You should feel free to request that in a PR.